### PR TITLE
Issue 1 user can request all characters

### PR DESCRIPTION
**Pull request by DreSean-Sutton**
image
image
Final-Project-issue-1

### CODE REVIEW

File: openapi.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                    rosterId:
+                      type: integer
+                    displayName:
+                      type: string
+                example:
+                -
+                  fighterId: 1
+                  fighter: banjo
+                  rosterId: 73
+                  displayName: banjo & kazooie
```
Comment (thebearingedge) [inline openapi.yml:50]: I would imagine that your display names should be properly title cased, no?

**Review by thebearingedge — CHANGES_REQUESTED** [review]
I added comments directly to your changes, but here are a couple of additional points

- please don't delete your package-lock file. run `npm install` and then be sure to add/commit your package-lock
- for the time being, it's a good idea to `npm uninstall` each package that you are not actually using.
- Until you support POST or PUT requests, there's no need for you to set up JSON middleware.
- I don't think static middleware is needed right now either.
- Please put your .env.example file back, it's there to let developers know what environment variables are needed for the app to run (but not their specific values). It's a template.
-

**Comment by DreSean-Sutton** [comment]
Closes #2

**Comment by DreSean-Sutton** [comment]
image
image
image

File: server/index.js [hunk 2]
```diff
@@ -1,27 +1,19 @@
...
 const pg = require('pg');
-// eslint-disable-next-line no-unused-vars
-const ClientError = require('./client-error');
-const errorMiddleware = require('./error-middleware');
-const staticMiddleware = require('./static-middleware');
-const pathToSwaggerUi = require('swagger-ui-dist').absolutePath();
-const cors = require('cors');
 const db = new pg.Pool({
+  const swaggerUi = require('swagger-ui-express');
+  const YAML = require('yamljs');
```
Comment (thebearingedge) [inline server/index.js]: These `require()` calls look out-of-place. I'd imagine that this crashes the server.

File: server/index.js [hunk 3]
```diff
@@ -36,7 +28,6 @@ app.get('/api/fighters', (req, res, next) => {
     })
     .catch(err => next(err));
 });
-app.use(errorMiddleware);
```
Comment (thebearingedge) [inline server/index.js:10]: You're going to want to leave the error middleware here.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, awesome, almost done!

**Pull request merged**